pkg/core/util.go

```diff
@@ -156,6 +156,6 @@ func calculateMultisigFee(n int) int64 {
 	bw := io.NewBufBinWriter()
 	emit.Int(bw.BinWriter, int64(n))
 	// it's a hack because prices of small PUSH* opcodes are equal
-	result = result + opcodePrice(opcode.Opcode(bw.Bytes()[0]))
+	result += opcodePrice(opcode.Opcode(bw.Bytes()[0]))
```

Comment:
Not a part of 3d04fd106d02ec8701f2740c6add15961cb2ead9.

Reply:
That was unsuccessful rebase. Fixed.